tests/tasks/setup/eks/awscli-cp-with-vpc.yaml

```diff
@@ -110,11 +110,10 @@ spec:
         }
       }' -n kube-system
       kubectl scale --replicas 1000 deploy coredns -n kube-system
-      #ToDo - remove these comments after experimentation
-      # Install EKS Pod Identity Agent 
-      # ENDPOINT_FLAG=""
-      # if [ -n "$(params.endpoint)" ]; then
-      #   ENDPOINT_FLAG="--endpoint $(params.endpoint)"
-      # fi
-      # aws eks $ENDPOINT_FLAG create-addon --cluster-name $(params.cluster-name) --addon-name eks-pod-identity-agent --addon-version v1.0.0-eksbuild.1
-      # aws eks $ENDPOINT_FLAG --region $(params.region) wait cluster-active --name $(params.cluster-name) 
\ No newline at end of file
+      # Install EKS Pod Identity Agent
+      ENDPOINT_FLAG=""
+      if [ -n "$(params.endpoint)" ]; then
+        ENDPOINT_FLAG="--endpoint $(params.endpoint)"
+      fi
+      aws eks $ENDPOINT_FLAG create-addon --cluster-name $(params.cluster-name) --addon-name eks-pod-identity-agent --addon-version v1.3.5-eksbuild.2
```

Comment:
let's not hardcode the addon version as its going to change. Let's take that as a param and have it defaulted to this.

Reply:
done